components/bookshelf/discover/SearchBox.tsx

```diff
@@ -33,17 +30,23 @@ const SearchBox = ({ setIsSearching }: SearchBoxProps) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
-  const handleChange = (e) => {
+  const handleChange = (e: {
+    target: { value: React.SetStateAction<string> }
+  }) => {
```

Comment:
![image](https://user-images.githubusercontent.com/780659/191928304-ce1dee7d-1281-4e7a-b2e6-f839176a8d49.png)

Reply:
Actually this type might be incorrect once you enable tailwind styled - typescript features